hw1/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New Jersey</title>
+</head>
+<body>
+    <div class="weather">
```

Comment:
It is important to use semantic tags. Try to replace divs with semantic tags were it will be appropriate.
For example, you can use `<header>`, `<footer>`, `<main>`.
 
https://www.freecodecamp.org/news/semantic-html5-elements/